src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.tizen.cs

```diff
@@ -0,0 +1,144 @@
+﻿using Microsoft.Maui.Platform;
+using Microsoft.Maui.Primitives;
+using Tizen.NUI;
+using Tizen.UIExtensions.NUI;
+using NHorizontalAlignment = Tizen.NUI.HorizontalAlignment;
+using NVerticalAlignment = Tizen.NUI.VerticalAlignment;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The native implementation of Popup control.
+/// </summary>
+public class MauiPopup : Popup
+{
+	readonly IMauiContext mauiContext;
+
+	/// <summary>
+	/// Constructor of <see cref="MauiPopup"/>.
+	/// </summary>
+	/// <param name="mauiContext">An instance of <see cref="IMauiContext"/>.</param>
+	/// <exception cref="ArgumentNullException">If <paramref name="mauiContext"/> is null an exception will be thrown. </exception>
+	public MauiPopup(IMauiContext mauiContext)
+	{
+		this.mauiContext = mauiContext ?? throw new ArgumentNullException(nameof(mauiContext));
+		OutsideClicked += OnOutsideClicked;
+	}
+
+	/// <summary>
+	/// An instance of the <see cref="IPopup"/>.
+	/// </summary>
+	public IPopup? VirtualView { get; private set; }
+
+	/// <inheritdoc/>
+	protected override void Dispose(bool isDisposing)
+	{
+		if (isDisposing)
+		{
+			OutsideClicked -= OnOutsideClicked;
+		}
+
+		base.Dispose(isDisposing);
+	}
+
+	/// <summary>
+	/// Method to initialize the native implementation.
+	/// </summary>
+	/// <param name="element">An instance of <see cref="IPopup"/>.</param>
+	public void SetElement(IPopup? element)
+	{
+		VirtualView = element;
+	}
+
+	/// <summary>
+	/// Method to show the Popup
+	/// </summary>
+	public void ShowPopup()
+	{
+		_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} cannot be null");
+		Content = VirtualView.Content?.ToPlatform(mauiContext) ?? throw new InvalidOperationException($"{nameof(VirtualView.Content)} cannot be null");
+
+		if (Content is null)
+		{
+			return;
+		}
```

Comment:
Do we need this check? If the Content is null we throw an exception at line 59. So already made the check, no?

Reply:
Done